doc-deploy-stable-index/actions.yml

```diff
@@ -0,0 +1,195 @@
+name: >
+  Doc deploy stable index action
+
+description: |
+  This action automates the process of creating indexes and scraping the
+  stable HTML documentation artifact that contains the development version
+  of a library, then deploying it to a Meilisearch instance.
+
+inputs:
+
+  # Required inputs
+
+  cname:
+    description: >
+      The Canonical Name (CNAME) that points to the documentation website.
+    required: true
+    type: string
+
+  library-name:
+    description: >
+      The name of the library.
+    required: true
+    type: string
+
+  api-key:
+    description: >
+      The API key used to access the Meilsearch instance host.
+    required: true
+    type: string
+
+  host-url:
+    description: >
+      The URL where the Meilsearch instance is hosted.
+    required: true
+    type: string
+
+  python-version:
+    description: >
+      Python version used for execution of the stable docs scraping.
+    required: true
+    type: string
+
+
+  # Optional inputs
+
+  doc-artifact-name:
+    description: >
+      The name of the HTML documentation artifact. This artifact is expected to
+      contain all the HTML and static files.The dafault value is ``documentation-html``.
+    required: false
+    default: 'documentation-html'
+    type: string
+
+  template:
+    description: >
+      The "template" parameter specifies the layout used for the HTML documentation.
+      By default, it is set to ``sphinx_pydata`` which assumes that the document is
+      constructed using the pydata-sphinx-theme or its associated theme,
+      such as the ansys-sphinx-theme.
+    required: false
+    default: "sphinx_pydata"
+    type: string
+
+  target:
+    description: >
+      Optional target used during the building process for version.
+    required: false
+    default: ''
+    type: string
```

Comment:
I can see the issue with the target. To simplify these actions, let's allow users to pass the name of the index. This simplifies a lot our action. This applies also to the dev action.